app/api-requests/business-eligible-to-apply.js

```diff
@@ -24,9 +25,13 @@ function applicationForBusinessInStateToApply (latestApplicationsForSbi) {
     console.log(`${new Date().toISOString()} Business is eligible to apply : ${JSON.stringify({
         sbi: latestApplication.data.organisation.sbi
       })}`)
-  // when toggle is on
-  } else if (latestApplication.statusId === status.AGREED && latestApplication.type === 'EE') {
-    throw new AlreadyAppliedError(`Business with SBI ${latestApplication.data.organisation.sbi} already has an endemics agreement`)
+  } else if (config.endemics.enabled) {
+    if (latestApplication.statusId === status.AGREED && latestApplication.type === 'EE') {
```

Comment:
What does **EE** refer to? we should add it to constant for better code readability